catalog/core/catalog-core-commands/pom.xml

```diff
@@ -129,10 +129,12 @@
                             ddf.catalog.data.impl,
                             ddf.catalog.operation.impl,
                             ddf.catalog.util.impl,
-                            ddf.catalog.filter.impl
+                            ddf.catalog.filter.impl,
+                            ddf.catalog.source.impl
                         </Private-Package>
                         <Export-Package />
                         <Import-Package>
```

Comment:
I believe I figured out how to not have this exclude here. We have some weird stuff going on. I can explain it better if we talk.